vulyk/control.py

```diff
@@ -227,4 +243,5 @@ def batch(batch_name, task_type):
         pt.add_row(values)
 
     print(pt)
+
```

Comment:
I use hot key `CTRL+ALT+I`  in PyCharm with default PEP8. 
In PEP8 we need to insert 2 spaces between functions and one space between class methods. Here we have many functions